connectors/bitbucket-server/src/test/resources/bitbucket/responses/success.json

```diff
@@ -77,9 +75,7 @@
       "creation_date": "1970-01-01T00:00:00Z",
       "header": {
         "title": "Bitbucket Server PR - APF-27-Test commit for stash connector./Sudharsan Sathiamoorthy",
```

Comment:
This just seems like too much information, formatted awkwardly (the title from the PR could have anything in it and can be very long), that will be in the body of the card.

The card is going to ellipsis this anyway, so you'll see something like "Bitbucket Server PR - APF-27-Test co...".

I recommend taking out the title + author from the card title.

Reply:
Removed title + author from the card header.